cmd/cli.go

```diff
@@ -254,81 +356,40 @@ var startTaskCmd = &cobra.Command{
 			return err
 		}
 
-		a := api.StartTaskArgs{
+		taskArgs := &task.StartTaskArgs{
 			Task: args[0],
-			ID:   args[1],
-		}
-
-		var resp api.StartTaskResp
-		err = cli.conn.Call("CedanaDaemon.StartTask", a, &resp)
-		if err != nil {
-			return err
+			// ID:   args[1],
```

Comment:
passing ID to startTask for args needs to come back - it's a simple way to attach a process to a jobID for an orchestrator